.github/workflows/e2e_custom_cl.yml

```diff
@@ -411,3 +415,7 @@ jobs:
           QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
           cache_key_id: solana-e2e-${{ env.MOD_CACHE_VERSION }}
           cache_restore_only: "false"
+        env:
+          E2E_TEST_CHAINLINK_IMAGE: ${{ env.CL_ECR }}          
+          E2E_TEST_SOLANA_SECRET: thisisatestingonlysecret
+          
```

Comment:
is there also a core side PR that updates the solana e2e in the `chainlink` repo?

otherwise updating the `chainlink-solana` ref in core may cause the solana e2e tests in there to break if these changes are not backwards compatible

Reply:
addressing this here: https://github.com/smartcontractkit/chainlink/pull/14332

since it's now blocking other e2e test updates